navigation/libandroid-navigation/src/main/res/layout/layout_notification_default.xml

```diff
@@ -12,49 +11,60 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:paddingTop="8dp"
-        android:paddingBottom="8dp">
+        android:orientation="horizontal">
 
         <ImageView
-            android:id="@+id/imagenotileft"
-            android:layout_width="wrap_content"
+            android:id="@+id/maneuverSignal"
+            android:layout_width="64dp"
             android:layout_height="match_parent"
-            android:layout_margin="16dp"
             android:contentDescription="@string/app_name"
-            android:src="@drawable/ic_maneuver_starting"
+            android:src="@drawable/ic_maneuver_arrive"
```

Comment:
I think it's better practice to load vector resources with `app:srcCompat` https://stackoverflow.com/questions/40624554/android-what-is-the-difference-between-appsrccompat-and-androidsrc

Reply:
Don't even need a resource here, accidentally added when designing the layout and I have now removed it.